src/main/java/github/weichware10/analyse/gui/admin/TrialCreator.java

```diff
@@ -3,6 +3,8 @@
 import github.weichware10.analyse.Main;
 import github.weichware10.util.Logger;
 import github.weichware10.util.gui.AbsScene;
+import github.weichware10.util.gui.Window;
+
 import java.io.BufferedWriter;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.io.BufferedWriter'